src/strategies/blacklist/index.ts

```diff
@@ -0,0 +1,38 @@
+import strategies from '..';
+
+export const author = 'joaomajesus';
+export const version = '1.0.0';
+
+let _options;
+
+function applyBlackList(address: string, result) {
+  return (_options.blacklist as Array<string>).includes(address) ? 0 : result;
```

Comment:
Can you make it work for both checksumed and not checksumed addresses.

Reply:
@bonustrack  I'll check that. 
I actually have another change in mind. so that the blacklisted addresses don't go through the decorated strategy

Reply:
sorry for my ignorance... :) but does that mean that I should use Address instead of string?

Reply:
@joaomajesus use below function:
```js
import { getAddress } from '@ethersproject/address';
const checksumedAddress = getAddress(address);
```
using this you can convert an address into a checksummed address

Reply:
@joaomajesus facing any issue? would love to help :)